AC3.2-MidUnit-6-Exam/DoubleVideoViewController.swift

```diff
@@ -7,25 +7,79 @@
 //
 
 import UIKit
+import AVFoundation
+import AVKit
+import MobileCoreServices
 
-class DoubleVideoViewController: UIViewController, CellTitled {
+
+class DoubleVideoViewController: UIViewController, CellTitled, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
     var titleForCell: String = "Double Video"
 
     @IBOutlet weak var videoContainerTop: UIView!
     @IBOutlet weak var videoContainerBottom: UIView!
     
+    var videoURL: URL?
+    
+    var topPlayer: AVPlayer?
+    var bottomPlayer: AVPlayer?
+    
+    var players: [AVPlayer?]!
+    var containers: [UIView]!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // don't f with this line
         self.edgesForExtendedLayout = UIRectEdge(rawValue: 0)
+        
+        self.players = [topPlayer, bottomPlayer]
+        self.containers = [videoContainerTop, videoContainerBottom]
     }
 
     override func viewDidLayoutSubviews() {
         // update your layers' frames here
+        // why?
```

Comment:
There was no need to do this because the project was set to only support portrait. Setting the frame of the layer once is ok. But to support rotation you could resize the frame of the layer here.